docs/getting-started/bug_disclosure_guidelines.md

```diff
@@ -12,12 +12,12 @@ Following [Google's standard disclosure policy](https://googleprojectzero.blogsp
 OSS-Fuzz will adhere to following disclosure principles:
 
   - **Deadline**. After notifying project authors, we will open reported
-    issues to the public in 90 days, or 30 days after the fix is released
-    (whichever comes earlier).
+    issues to the public in 90 days, or after the fix is released (whichever
```

Comment:
nit: maybe say "immediately after the fix is released"
Current wording is ambiguous.

Reply:
"Immediately" is also inaccurate (we can be delayed by a day or so), so keeping the current wording.